src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -9,7 +9,7 @@
     tabindex="0"
   >
     <div
-      v-if="isSearchable && !isSingleLine"
+      v-if="isSearchable && !props.isSingleLine"
```

Comment:
All places where you're utilizing props in the `template` section: Since props are defined down in the `script setup` function, you don't need to reference them with `props.*` in the template (and actually, you shouldn't).

They are automatically exposed as the prop name, e.g. `isSingleLine`

Reply:
Ha, interesting. I added those references specifically because I believe in the opposite: props should always be referred to using `props.` because that makes it unambiguous where they come from.

Reply:
I’ve removed all instances of this now. I still think this is worse because it’s now more ambiguous where symbols come from, but I’m fine enough with not doing it.

Reply:
I understand well _how_ this work and which things are exposed in this manner and which aren’t.

My point is that I think this is a unnecessarily ambiguous way of referencing symbols in the template. Personally, I think referring to `props.showCopyButton` is **far** more clear than just `showCopyButton`: In other words, I would actually _disallow_ referring to props without accessing the `props` object.

But as I said, I’m fine with this stance as well. :)

Reply:
Last comment for me 😬 

> Declared props are automatically exposed to the template.
> [Source](https://vuejs.org/guide/essentials/component-basics.html#passing-props)

But yes I don't disagree at all about it being more readable.